packages/desktop-client/src/components/schedules/index.js

```diff
@@ -83,7 +83,6 @@ export default function Schedules() {
           allowCompleted={true}
           onSelect={onEdit}
           onAction={onAction}
-          style={{ backgroundColor: 'white' }}
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** ‏oops?

Before: empty schedule table has white background

After: empty schedule table has gray-ish background